test/fuzz/signature/signature.go

```diff
@@ -57,3 +60,88 @@ func FuzzECDSASigner(data []byte) int {
 
 	return 1
 }
+
+func FuzzComputeDigest(data []byte) int {
+	ecdsaSupportedHashFuncs := []crypto.Hash{
```

Comment:
maybe a more generic name since there's no relation to ecdsa here?

Reply:
Fixed it.